diff --git a/app/models/macro.rb b/app/models/macro.rb
index fc25bee..bc30d63 100644
--- a/app/models/macro.rb
+++ b/app/models/macro.rb
@@ -32,7 +32,8 @@ class Macro < ApplicationRecord
 
   ACTIONS_ATTRS = %w[send_message add_label assign_team assign_agent mute_conversation change_status remove_label remove_assigned_agent
                      remove_assigned_team resolve_conversation snooze_conversation change_priority send_email_transcript
-                     send_attachment add_private_note send_webhook_event].freeze
+                     send_attachment add_private_note send_webhook_event add_to_kanban_board move_kanban_card
+                     assign_kanban_card].freeze
 
   def set_visibility(user, params)
     self.visibility = params[:visibility]
