diff --git a/app/javascript/dashboard/routes/dashboard/settings/macros/MacroEditor.vue b/app/javascript/dashboard/routes/dashboard/settings/macros/MacroEditor.vue
index 1312122..0d968fc 100644
--- a/app/javascript/dashboard/routes/dashboard/settings/macros/MacroEditor.vue
+++ b/app/javascript/dashboard/routes/dashboard/settings/macros/MacroEditor.vue
@@ -41,12 +38,12 @@ const isPublicMacroReadOnly = computed(
   () => macro.value?.visibility === 'global' && !isAdmin.value
 );
 
-const fetchDropdownData = () =>
-  Promise.all([
-    store.dispatch('agents/get'),
-    store.dispatch('teams/get'),
-    store.dispatch('labels/get'),
-  ]);
+const fetchDropdownData = () => {
+  store.dispatch('agents/get');
+  store.dispatch('teams/get');
+  store.dispatch('labels/get');
+  store.dispatch('kanbanBoards/fetchBoards');
+};
 
 const formatMacro = macroData => {
   const formattedActions = macroData.actions.map(action => {
