diff --git a/app/javascript/dashboard/routes/dashboard/settings/macros/constants.js b/app/javascript/dashboard/routes/dashboard/settings/macros/constants.js
index 8a0088d..71597ca 100644
--- a/app/javascript/dashboard/routes/dashboard/settings/macros/constants.js
+++ b/app/javascript/dashboard/routes/dashboard/settings/macros/constants.js
@@ -69,6 +69,21 @@ export const MACRO_ACTION_TYPES = [
     label: 'CHANGE_PRIORITY',
     inputType: 'search_select',
   },
+  {
+    key: 'add_to_kanban_board',
+    label: 'ADD_TO_KANBAN_BOARD',
+    inputType: 'kanban_stage',
+  },
+  {
+    key: 'move_kanban_card',
+    label: 'MOVE_KANBAN_CARD',
+    inputType: 'kanban_stage',
+  },
+  {
+    key: 'assign_kanban_card',
+    label: 'ASSIGN_KANBAN_CARD',
+    inputType: 'kanban_agents',
+  },
   {
     key: 'send_webhook_event',
     label: 'SEND_WEBHOOK_EVENT',
