diff --git a/app/controllers/api/v1/accounts/macros_controller.rb b/app/controllers/api/v1/accounts/macros_controller.rb
index c4e0cd6..7f7b2c0 100644
--- a/app/controllers/api/v1/accounts/macros_controller.rb
+++ b/app/controllers/api/v1/accounts/macros_controller.rb
@@ -58,7 +58,11 @@ class Api::V1::Accounts::MacrosController < Api::V1::Accounts::BaseController
   def permitted_params
     params.permit(
       :name, :visibility,
-      actions: [:action_name, { action_params: [] }]
+      actions: [
+        :action_name,
+        { action_params: [:kanban_board_id, :kanban_stage_id, :message, { agent_ids: [], team_ids: [] }] },
+        { action_params: [] }
+      ]
     )
   end
 
