diff --git a/app/javascript/dashboard/routes/dashboard/dashboard.routes.js b/app/javascript/dashboard/routes/dashboard/dashboard.routes.js
index a4acf76..639d90a 100644
--- a/app/javascript/dashboard/routes/dashboard/dashboard.routes.js
+++ b/app/javascript/dashboard/routes/dashboard/dashboard.routes.js
@@ -1,10 +1,11 @@
 import settings from './settings/settings.routes';
 import conversation from './conversation/conversation.routes';
 import { routes as searchRoutes } from '../../modules/search/search.routes';
-import { routes as callRoutes } from './calls/routes';
 import { routes as contactRoutes } from './contacts/routes';
 import { routes as companyRoutes } from './companies/routes';
+import { routes as notificationRoutes } from './notifications/routes';
 import { routes as inboxRoutes } from './inbox/routes';
+import { routes as kanbanRoutes } from './kanban/routes';
 import { frontendURL } from '../../helper/URLHelper';
 import helpcenterRoutes from './helpcenter/helpcenter.routes';
 import campaignsRoutes from './campaigns/campaigns.routes';
@@ -25,10 +25,11 @@ export default {
         ...inboxRoutes,
         ...conversation.routes,
         ...settings.routes,
-        ...callRoutes,
         ...contactRoutes,
         ...companyRoutes,
+        ...kanbanRoutes,
         ...searchRoutes,
+        ...notificationRoutes,
         ...helpcenterRoutes.routes,
         ...campaignsRoutes.routes,
       ],
