diff --git a/app/models/conversation.rb b/app/models/conversation.rb
index 62cbeaa..08820f9 100644
--- a/app/models/conversation.rb
+++ b/app/models/conversation.rb
@@ -132,6 +132,7 @@ class Conversation < ApplicationRecord
   has_many :attachments, through: :messages
   has_many :reporting_events, dependent: :destroy_async
   has_many :automation_rule_pending_executions, dependent: :delete_all
+  has_many :kanban_cards, dependent: nil
 
   before_save :ensure_snooze_until_reset
   before_save :set_status_changed_at
