diff --git a/app/models/account.rb b/app/models/account.rb
index 8126081..3baeab6 100644
--- a/app/models/account.rb
+++ b/app/models/account.rb
@@ -104,6 +104,8 @@ class Account < ApplicationRecord
   has_many :webhooks, dependent: :destroy_async
   has_many :whatsapp_channels, dependent: :destroy_async, class_name: '::Channel::Whatsapp'
   has_many :working_hours, dependent: :destroy_async
+  has_many :kanban_automation_rules, dependent: :destroy_async
+  has_many :kanban_automation_logs, dependent: :destroy_async
 
   has_one_attached :contacts_export
 
