diff --git a/app/javascript/dashboard/helper/URLHelper.js b/app/javascript/dashboard/helper/URLHelper.js
index 8437e11..774c0aa 100644
--- a/app/javascript/dashboard/helper/URLHelper.js
+++ b/app/javascript/dashboard/helper/URLHelper.js
@@ -160,3 +160,6 @@ export const sanitizeAllowedDomains = domains => {
 
   return unique.join(',');
 };
+
+export const kanbanConversationUrl = ({ accountId, boardId, conversationId }) =>
+  `accounts/${accountId}/kanban/${boardId}/conversations/${conversationId}`;
