diff --git a/app/javascript/shared/constants/busEvents.js b/app/javascript/shared/constants/busEvents.js
index deb07ee..8f2b062 100644
--- a/app/javascript/shared/constants/busEvents.js
+++ b/app/javascript/shared/constants/busEvents.js
@@ -13,4 +13,5 @@ export const BUS_EVENTS = {
   NEW_CONVERSATION_MODAL: 'newConversationModal',
   INSERT_INTO_RICH_EDITOR: 'insertIntoRichEditor',
   MFA_STATE_CHANGED: 'MFA_STATE_CHANGED',
+  KANBAN_REALTIME_EVENT: 'KANBAN_REALTIME_EVENT',
 };
